projects/security.md

```diff
@@ -0,0 +1,58 @@
+# Security Semantic Conventions Working Group
+
+## Description
+
+The purpose of this working group is to bring in the security domain for the OpenTelemetry community. 
+
+As outlined in the [ECS OTEP](https://github.com/open-telemetry/oteps/blob/main/text/0199-support-elastic-common-schema-in-opentelemetry.md), the Elastic Common Schema (ECS) is currently being contributed to the semantic conventions schema. Given the significance of security within ECS, establishing this working group is crucial as it will expedite the donation of ECS fields tailored to security use cases. Beyond expanding the schema, our aim is to craft a clear vision for the instrumentation required.
+
+## Deliverables
+
+* Our current focus is on defining essential semantic conventions for security use cases. 
+* As new use cases and namespaces are introduced to the semantic conventions, there may be a need for additional instrumentation to accommodate them. It is anticipated that this aspect will expand through an iterative process.
+
+## Staffing / Help Wanted
+
+We are seeking security experts to collaborate with us in expanding the security domain within the community.  
+
+### Required staffing
+
+There is an open [PR](https://github.com/open-telemetry/semantic-conventions/issues/580) to create a `semconv-security-approver` group for all PRs related to security fields.  
+
+* project lead: @trisch-me
+* domain expert: @mjwolf
+
+* GC sponsor: @reyang
+* GC sponsor: vacant
```

Comment:
```suggestion
* TC sponsor: @reyang
* TC sponsor: vacant
```

Reply:
Do you think it's possible to change/ upgrade name of existing group?
If we are going to make security the second big domain within otel on the same level as O11y it makes sense to have this group to have `Security` name and the other group to have a new one like `Applied Security` or so